packages/cli/src/bin/index.ts

```diff
@@ -173,6 +181,15 @@ program
     `${defaultWebUIPort}`,
   )
   .action(async (options) => {
+
+    // Check if ui command is executed before the run command. 
+    try {
+        await fs.readFile(outputFilePath);
+      } catch (err) {
+        console.log(buildErrorLog(`Failed to read output file`));
```

Comment:
I think this message is not that useful for the user. Can directly say what action to take.

Reply:
Fixed!